templates/books/book-detail.html

```diff
@@ -84,11 +84,6 @@ <h1 class="h2">{% dtranslate book.title %}</h1>
                 {% endif %}
                 {% cite_source book.description_source "cit-description" %}
             </div>
-
-            {% if book.preview_url %}
-                {% translate "Першую частку можна паслухаць бясплатна на" %} <a href="{{ book.preview_url }}" target="_blank">YouTube</a>.
-            {% endif %}
```

Comment:
I need to make sure I ran the migration and fix the cases with 2+ narrations before deploying this, right?

Reply:
of course

Reply:
Sounds good.